docs/hazmat/primitives/symmetric-encryption.rst

```diff
@@ -147,12 +147,12 @@ Algorithms
         :term:`bits` (32 bytes) in length.
     :type key: :term:`bytes-like`
 
-    :param nonce: Should be unique, a :term:`nonce`. It is
-        critical to never reuse a ``nonce`` with a given key.  Any reuse of a
-        nonce with the same key compromises the security of every message
-        encrypted with that key. The nonce does not need to be kept secret
-        and may be included with the ciphertext. This must be ``128``
-        :term:`bits` in length.
+    :param nonce: In our context, the 128-bit parameter ``nonce`` is a concatenation
```

Comment:
The way this is phrased now is a bit confusing and puts implementation details before the security instructions.

My recommendation is to keep the existing text and then add a description that the 16-bytes are composed of the counter and underlying nonce.

Reply:
I agree with you, so I made suggested changes in the new commit.